src/uct/ib/rc/base/rc_iface.c

```diff
@@ -966,3 +971,30 @@ void uct_rc_iface_vfs_refresh(uct_iface_h iface)
         rc_iface_ops->ep_vfs_populate(ep);
     }
 }
+
+static void ucp_send_op_mpool_obj_str(ucs_mpool_t *mp, void *obj,
+                                      ucs_string_buffer_t *strb)
+{
+    uct_rc_iface_send_op_t *op = obj;
+    const char *handler_func_name;
+    const char *comp_func_name;
+
+    ucs_string_buffer_appendf(strb, "flags:%x ", op->flags);
+
+    handler_func_name = ucs_debug_get_symbol_name(op->handler);
+    ucs_string_buffer_appendf(strb, "handler:%s()", handler_func_name);
```

Comment:
maybe unify with line 982 to single appendf() call?

Reply:
yep, unified